docs/tutorials/complete_applications/complete_application/step_04/step_04.md

```diff
@@ -0,0 +1,181 @@
+> The full code is available
+<a href="./../src/src.zip" download>here</a>.
+
+The Scenario Page is a part of the application designed for creating and 
+customizing prediction scenarios using time series data. Users can modify different parameters 
+for the prediction, such as the prediction date, maximum capacity, 
+and the number of predictions. This page also includes a chart displaying historical data 
+and predictions generated using both machine learning and baseline methods.
+
+![Scenario Page](result.png){ width=700 style="margin:auto;display:block;border: 4px solid rgb(210,210,210);border-radius:7px" }"
+
+The Scenario Page is constructed using a combination of Markdown and Python code. Below is a detailed explanation of each component:
+
+# Markdown (pages/scenario/scenario.md)
+
+```markdown
+# Create your scenario:
+
+<|layout|columns=3 1 1 1 1|
+<|{scenario}|scenario_selector|>
+
+**Prediction date** <br/>
+<|{day}|date|active={scenario}|not with_time|>
+
+**Max capacity** <br/>
+<|{max_capacity}|number|active={scenario}|>
+
+**Number of predictions** <br/>
+<|{n_predictions}|number|active={scenario}|>
+
+<br/> <|Save|button|on_action=save|active={scenario}|>
+|>
+ 
+<|{scenario}|scenario|on_submission_change=submission_change|>
+
+<|{predictions_dataset}|chart|x=Date|y[1]=Historical values|type[1]=bar|y[2]=Predicted values ML|y[3]=Predicted values Baseline|>
+
+# Data Node Exploration
+
+<|layout|columns=1 5|
+<|{data_node}|data_node_selector|>
+
+<|{data_node}|data_node|>
+|>
+```
+
+The Markdown section outlines the arrangement and elements of the Scenario Page. It consists of the following components:
+
+- **Scenario Selector**: `<|{scenario}|scenario_selector|>`
+
+A component (a dropdown) that allows users to select different scenarios. The selected scenario will affect the values of other components on the page.
```

Comment:
Should we say something like this?
The scenario value is used in other elemtns so selecting a scenario affects the other elements.

Reply:
I added your sentence, this is a good idea